tgui/packages/tgui/interfaces/Pod/Parts/CargoHold.tsx

```diff
@@ -0,0 +1,40 @@
+import { useBackend } from '../../../backend';
+import { Box, Button, NoticeBox, Section } from '../../../components';
+type Data = {
+  storedName: string;
+  ref: string;
+};
+
+export default function CargoHold(props: { partData: Data }): JSX.Element {
+  const { act } = useBackend<{
+    ourData: Data;
+  }>();
+  const ourData = props.partData as Data;
```

Comment:
This seems inaccurate, data comes from the backend, but partdata is coming from props?

If ourData is in props you shouldn't need to coerce it with as, the props type should have this

Pls look at other examples where they're assigning a type to data and props, separately